### PR TITLE
updated font weight for article summary headline - ios platform specific

### PR DESCRIPTION
**Pull request by qalvapps**

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/4eb8f0b3fcd3b3f85b351e96aa208e6b6d5dabad/dextrose-test/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/918f68aa98a8480d40833aec007d301ff2c89b73/dextrose-test/index.html

**Comment by nikhedonia** [comment]
`react-native-web` assumes files with no suffix either as generic or as ios specific.
If you require specific behaviour for each platform, use the suffixes `web` and `android` and no suffix for ios.

**Comment by colincclark** [comment]
@nikhedonia nice investigation...horrible result....RNW strikes again

**Comment by qalvapps** [comment]
following on from gaetano's conclusion ... i'm even more confused (didn't think that was possible)  should I rename my style sheet override ? and to what ? waaaah !

**Comment by nikhedonia** [comment]
> I rename my style sheet override
I've already renamed the stylesheets; Pull and have a look at my commit.
Btw kudos go to @craigbilner  who was able to pinpoint the issue in an instant

File: packages/article-summary/styles/index.js [hunk 1]
```diff
@@ -1,3 +1,11 @@
 import sharedStyles from "./shared";
 
-export default sharedStyles;
+const styles = {
+  ...sharedStyles,
+  headline: {
+    ...sharedStyles.headline,
+    fontWeight: "900"
```
Comment (colincclark) [inline packages/article-summary/styles/index.js]: this changes `fontWeight` for all devices and web, was that the intention?

Comment (nikhedonia) [reply packages/article-summary/styles/index.js]: No : 

index.js => ios
index.web.js => web
index.android.js => android

Comment (nikhedonia) [reply packages/article-summary/styles/index.js]: index.web.js now just returns the shared style

Comment (colincclark) [reply packages/article-summary/styles/index.js]: that's confusing, but ok....

Comment (colincclark) [reply packages/article-summary/styles/index.js]: this just needs to be formatted now so that the linter passes

Comment (aronshamash) [reply packages/article-summary/styles/index.js]: @nikhedonia it is not possible to use `index.ios.js` for ios and have no `index.js` file ?

Comment (aronshamash) [reply packages/article-summary/styles/index.js]: that would be less confusing..

Comment (colincclark) [reply packages/article-summary/styles/index.js]: @aronshamash `ios.js` are not picked up by RNW

Comment (craigbilner) [reply packages/article-summary/styles/index.js]: from a jest perspective it's iOS first sadly

**Pull request merged**